ProjectCoimbra.UWP/Project.Coimbra/Pages/SongPage.xaml.cs

```diff
@@ -46,5 +48,34 @@ private async void NextButton_Click(object sender, RoutedEventArgs e)
 
             this.Next.IsEnabled = false;
         }
+
+        private async void Remove_Item(object sender, RoutedEventArgs e)
+        {
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = "Remove Song from List",
+                Content = "Are you sure you want to delete this song from your list?",
+                PrimaryButtonText = "Yes",
+                CloseButtonText = "Cancel"
+            };
+
+            var result = await dialog.ShowAsync();
+
+            if (result == ContentDialogResult.Primary)
+            {
+                var item = (KeyValuePair<string, string>)((FrameworkElement)sender).DataContext;
+
+                try
+                {
+                    await SongPagesHelper.RemoveFile(item.Key);
+                }
+                catch (Exception)
+                {
+                    return;
```

Comment:
You should never suppress all exceptions as it is a security risk. Instead, catch the exceptions you can reasonably handle